### PR TITLE
Subscription variant selector and Cart item with information of subscription

### PR DESCRIPTION
**Pull request by pebiantara**

### CODE REVIEW

File: src/scss/components/_product-variant.scss [hunk 1]
```diff
@@ -24,13 +25,41 @@
...
-			.product-swatch {
+			.product-swatch,
+			.product-variant__note {
 				display: block;
 			}
 		}
 	}
 
+	&--subscription {
+		.subscription-title {
```
Comment (foria) [inline src/scss/components/_product-variant.scss:36]: please use BEM convention, it must be subscription__title

File: src/js/modules/products.js [hunk 2]
```diff
@@ -155,8 +155,27 @@ const productWaitlistSubmitted = $('.product-waitlist__submitted');
...
+		$('[data-toggle="popover"]').popover({
+			delay: {
+				show: 100,
+			},
+		});
+	});
+}
+
+// Dismissable popover click out side
+$('body').on('click', function (e) {
```
Comment (foria) [inline src/js/modules/products.js:175]: this function should be run only if the popover element is in the page and open

**Review by foria — APPROVED** [review]

**Review by foria — CHANGES_REQUESTED** [review]
check my comments

File: src/scss/components/_product-variant.scss (hunk 1 above)
Comment (pebiantara) [reply src/scss/components/_product-variant.scss:36]: Ops let me update

File: src/js/modules/products.js (hunk 2 above)
Comment (pebiantara) [reply src/js/modules/products.js:175]: sure okay

File: src/scss/components/_product-variant.scss (hunk 1 above)
Comment (pebiantara) [reply src/scss/components/_product-variant.scss]: Updated

File: src/js/modules/products.js (hunk 2 above)
Comment (pebiantara) [reply src/js/modules/products.js]: Updated

**Comment by pebiantara** [comment]
hi @foria i have updated please check again

**Review by foria — APPROVED** [review]

**Pull request merged**